### PR TITLE
Run3-ft20 Modify eras and cff's to accommodate steps using dd4hep

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Modify eras and cff's to accommodate steps using dd4hep

#### PR validation:

Use runTheMatrix test workflows

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Configuration/Eras/python/Era_Run3_dd4hep_cff.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run3_cff import Run3
+from Configuration.ProcessModifiers.dd4hep_cff import dd4hep
+
+Run3_dd4hep = cms.ModifierChain(Run3, dd4hep)
```
Comment (kpedro88) [inline Configuration/Eras/python/Era_Run3_dd4hep_cff.py:6]: please add this in Configuration/StandardSequences/python/Eras.py

Comment (bsunanda) [reply Configuration/Eras/python/Era_Run3_dd4hep_cff.py:6]: Done

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
+1

**Comment by bsunanda** [comment]
@kpedro88 Could you sign this please?

**Comment by kpedro88** [comment]
+upgrade

**Comment by bsunanda** [comment]
@silviodonato Can this PR be integrated because that will help testing dd4hep definitions for Run3 geometry

**Comment by bsunanda** [comment]
@civanch Could you make this Urgent as well. Without this one cannot test dd4hep applications

**Comment by cvuosalo** [comment]
urgent

**Comment by cvuosalo** [comment]
@bsunanda @civanch I made this PR urgent

**Comment by silviodonato** [comment]
+operations

**Comment by silviodonato** [comment]
+1

**Pull request merged**